examples/ZMP/LinearInvertedPendulum.m

```diff
@@ -292,11 +292,7 @@
           s2traj = PPTrajectory(pchipDeriv(breaks,fliplr(y.'),fliplr(ydot.')));
           Vt = QuadraticLyapunovFunction(getInputFrame(ct),S,s1traj,s2traj);
         else
-          s1traj = struct('breaks', breaks, ...
-                          'K', eye(4), ...
-                          'A', A2,...
-                          'alpha', alpha,...
-                          'gamma', beta);
+          s1traj = ExpPlusPPTrajectory(breaks,eye(4),A2,alpha,beta);
```

Comment:
It is. I'd hacked this to not actually construct the trajectory when it was part of the recovery planning loop. Doing so made life difficult in a bunch of places where we had to check if we actually had a trajectory or not, and this is no longer part of the recovery planner, so I reverted it.